gmso/tests/test_convert_networkx.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import unyt as u
+
+import gmso
+from gmso.core.topology import Topology as Top
+from gmso.core.subtopology import SubTopology as SubTop
+from gmso.core.atom import Atom
+from gmso.external.convert_networkx import from_networkx, to_networkx
+from gmso.tests.base_test import BaseTest
+
+class TestConvertNetworkX(BaseTest):
+    def test_to_networkx_ethane(self, ethane):
+        ethane_to_nx = to_networkx(ethane)
+
+        assert ethane.n_sites == ethane_to_nx.number_of_nodes()
```

Comment:
these are great tests, if we can add in the possibilities above, this is good to go afterwards